csi/moac/nexus.ts

```diff
@@ -299,8 +311,14 @@ export class Nexus {
   // Destroy nexus on storage node.
   async destroy() {
     log.debug(`Destroying nexus "${this}" ...`);
-    await this.node.call('destroyNexus', { uuid: this.uuid });
-    log.info(`Destroyed nexus "${this}"`);
+    if (!this.node!.isSynced()) {
+      // We don't want to block the volume life-cycle in case that the node
+      // is down - it may never come back online.
+      log.warn(`Faking the destroy of "${this}" because it is unreachable`);
```

Comment:
Am I right in thinking that if we fake the destruction of the nexus and it reappears somehow (maybe due to a transient network fault) that the FSA loop will eventually destroy it for real?

Reply:
Correct. And for the well known problem of queued IOs being written to replicas from reappearing nexus we must use ACLs or similar.

Reply:
Thanks :+1: